c++/src/capnp/fuzz-convert.c++

```diff
@@ -0,0 +1,29 @@
+#include <fcntl.h>
+
+#include <kj/io.h>
+#include <kj/main.h>
+#include "message.h"
+#include "serialize.h"
+#include "serialize-packed.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+    kj::ArrayPtr<const uint8_t> array(Data, Size);
+    kj::ArrayInputStream ais(array);
+
+    capnp::ReaderOptions options;
+
+    //TODO use readOneAndConvert(input, output) ?
+    //with int fd = open("/dev/null", O_WRONLY);
+    //and kj::FdOutputStream output(fd);
+
+    KJ_IF_MAYBE(e, kj::runCatchingExceptions([&]() {
+        kj::BufferedInputStreamWrapper input(ais);
+        capnp::InputStreamMessageReader message(input, options);
+        capnp::PackedMessageReader messagep(input, options);
```

Comment:
This test case seems kind of uninteresting as it isn't actually invoking any capnp parsing code -- it's just setting up a `MessageReader` and then throwing it away. Since it's a `PackedMessageReader`, I suppose it does exercise some of the "packed" format decoder, but if that's the goal it might make sense to target `PackedInputStream` more directly. Otherwise, I think I'd just delete `fuzz-convert.c++`.

Reply:
Feel free to delete it.

This was inspired by `readOneAndConvert` which does
```
      case Format::BINARY: {
        capnp::InputStreamMessageReader message(input, options);
        return writeConversion(message.getRoot<AnyStruct>(), output);
      }
      case Format::PACKED: {
        capnp::PackedMessageReader message(input, options);
        return writeConversion(message.getRoot<AnyStruct>(), output);
      }
```

Reply:
Yes, `readOneAndConvert()`, when implementing `capnp convert packed:binary`, does not actually do very much.

Let's remove `fuzz-convert.c++`.

Reply:
done